internal/api/v1/stage.go

```diff
@@ -157,18 +165,18 @@ func (hc ApplicationsController) Stage(w http.ResponseWriter, r *http.Request) A
 		deploymentImageURL = gitea.LocalRegistry
 	}
 
-	pr := newPipelineRun(uid, app, mainDomain, registryURL, deploymentImageURL)
+	pr := newPipelineRun(uid, params, mainDomain, registryURL, deploymentImageURL)
 	o, err := client.Create(ctx, pr, metav1.CreateOptions{})
 	if err != nil {
 		return InternalError(err, fmt.Sprintf("failed to create pipeline run: %#v", o))
 	}
 
-	err = auth.CreateCertificate(ctx, cluster.RestConfig, app.Name, app.Org, mainDomain)
+	err = auth.CreateCertificate(ctx, cluster.RestConfig, params.Name, params.Org, mainDomain)
```

Comment:
Shouldn't this certificate have an owner too?

Reply:
Oh yes!